autogen/agentchat/groupchat.py

```diff
@@ -448,3 +454,55 @@ def _raise_exception_on_async_reply_functions(self) -> None:
 
         for agent in self._groupchat.agents:
             agent._raise_exception_on_async_reply_functions()
+
+    def clear_agents_history(self, reply: str, groupchat: GroupChat) -> str:
+        """Clears history of messages for all agents or selected one. Can preserve selected number of last messages.
+
+        Args:
+            reply (str): Admin reply to analyse.
```

Comment:
Add examples and explanations about the expected behaviors.

Reply:
Done